packages/@aws-cdk/aws-backup/lib/plan.ts

```diff
@@ -28,6 +28,15 @@ export interface BackupPlanProps {
    */
   readonly backupPlanName?: string;
 
+  /**
+   * The option for backup.
+   *
+   * @default - Object used to set Windows VSS options.
+   * @see {@link https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-backup-backupplan-windows.html}
```

Comment:
dead link, should be https://docs.aws.amazon.com/aws-backup/latest/devguide/windows-backups.html